<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamControl</class>
 <widget class="QWidget" name="TeamControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>队伍信息管理系统</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>721</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="sortbyBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作品名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学校名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>赛事类别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>参赛者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>指导老师</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="orderBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>降序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/sort.png</normaloff>:/new/prefix1/image/sort.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="idlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/add.png</normaloff>:/new/prefix1/image/add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/delete.png</normaloff>:/new/prefix1/image/delete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/update.png</normaloff>:/new/prefix1/image/update.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="optionBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作品名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学校名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>赛事类别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>参赛选手</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>指导老师</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;查找选项：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contentEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;查找内容：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serachButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/filter.png</normaloff>:/new/prefix1/image/filter.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/refresh.png</normaloff>:/new/prefix1/image/refresh.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/input.png</normaloff>:/new/prefix1/image/input.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/image/output.png</normaloff>:/new/prefix1/image/output.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
